package.json

```diff
@@ -3,7 +3,7 @@
   "description": "React Native module for Radar, the leading geofencing and location tracking platform",
   "homepage": "https://radar.com",
   "license": "Apache-2.0",
-  "version": "3.7.5",
+  "version": "3.7.6",
```

Comment:
A few more versions to bump, but I assume you're waiting on the android release first.

https://www.notion.so/radarlabs/How-to-release-the-React-Native-SDK-21c70f16e0bd4fe28425923645cadda0

Reply:
Yea — there's now 3 PRs open for RN that we're trying to get in. @nickpatrick 's native version bump, this, and the UI kit. Was just gonna handle the GH workflow stuff in this given all the stuff in flight.